amm/interp/src/main/scala/ammonite/runtime/Interpreter.scala

```diff
@@ -699,9 +700,15 @@ object Interpreter{
     Name(wrapperName.raw + (if (wrapperIndex == 1) "" else "_" + wrapperIndex))
   }
 
-  def initPrinters(output: OutputStream, error: OutputStream) = {
+  def initPrinters(output: OutputStream, error: OutputStream, verboseOutput: Boolean) = {
     val colors = Ref[Colors](Colors.Default)
-    val printStream = new PrintStream(output, true)
+    val printStream = verboseOutput match{
+      case true => new PrintStream(output, true)
+      //make a dummy printstream in case of non-verbose output
+      case false => new PrintStream(new OutputStream(){
+        def write(i: Int) = ()
+      })
+    }
```

Comment:
Yeah that's probably fine; I don't think a blanket "shut off all stdout" is something people would expect, so instead having it "shut off some subset of auxiliary output we deem unimportant" might be easier to understand. Then we can turn it on case by case without worrying about weird interactions